cli/tests/pcluster/templates/test_additional_packages.py

```diff
@@ -65,3 +65,55 @@ def test_intel_hpc_platform(mocker, test_datadir, config_file_name, enabled):
             assert_that(tag["Value"]).is_equal_to("enable_intel_hpc_platform=true")
         else:
             assert_that(tag).is_none()
+
+
+@pytest.mark.parametrize("enabled", [True, False])
+def test_intel_one_api_toolkits(mocker, test_datadir, enabled, pcluster_config_reader):
+    """Test the output template has the right root volume size, timeout, dna.json according to the input config file."""
+    mock_aws_api(mocker)
+    mock_bucket_object_utils(mocker)
+
+    input_yaml = load_yaml_dict(pcluster_config_reader(enabled=enabled))
+
+    cluster_config = ClusterSchema(cluster_name="clustername").load(input_yaml)
+
+    generated_template, _ = CDKTemplateBuilder().build_cluster_template(
+        cluster_config=cluster_config, bucket=dummy_cluster_bucket(), stack_name="clustername"
+    )
+
+    resources = generated_template["Resources"]
+    head_node_launch_template = resources["HeadNodeLaunchTemplate"]
+
+    # Check root volume size
+    block_device_mappings = head_node_launch_template["Properties"]["LaunchTemplateData"]["BlockDeviceMappings"]
```

Comment:
Minor: we have an utility function [`get_resources`](https://github.com/aws/aws-parallelcluster/blob/e33258ca03f5c76239df8464cda7e12fa07672c2/cli/tests/pcluster/utils.py#L39-L55) to inspect resources in the generated template that allows you to filter them. In this simple case you only filter by name, so not a big deal, but just to let you know.